PROJECT_PROPOSAL.md

```diff
@@ -0,0 +1,95 @@
+# Concrete user interfaces for abstract algebra
+
+## Problem
+
+A "proof" is an argument considered convincing by some community of practice, like plumbers or mathematicians. A "formal system" is a rulebook declaring precisely which proofs are considered convincing. "Algebra" is hard to define, but you know it when you see it.
+
+Mathematical formal systems have exploded in breadth and scope over the past century. Interest in "mathematical foundations" -- a ruleset for all of mathematics -- has remained keen since the notion was first conceived, despite its being proven impossible. In the past few decades, uptake of computer-checked formal proof assistants has grown, with substantial bodies of work being formalized. Much of this work has an algebraic and constructive flavor -- it mainly involves concrete manipulations of tree-like data structures.
+
+The public has no idea that any of this is going on. In fact they have little understanding of what mathematicians do at all. There *is* public interest in mathematics, as demonstrated by the popularity of events like the "Summers of Math Exposition" on YouTube. However, most of the infrastructure of contemporary mathematics remains out-of-reach except to experts.
+
+This might be surprising to people working in these fields. Contemporary algebra, and "higher category theory" especially, tend to build on simple and intuitive spatial metaphors. For example:
+- Rearranging trees (traditional algebra)
+- Gluing together triangles (commutative diagrams)
+- Sliding around beads on tangled wires (string diagrams)
+And so on. How could anybody not understand these ideas?
```

Comment:
Probably I should have citations here but I have no idea what to cite. Maybe a higher category theory survey?